ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -15,7 +15,7 @@ cluster "aws" {
   os_channel   = var.os_channel
   ssh_pubkeys  = ["$PUB_KEY"]
 
-  worker_pool "$CLUSTER_ID-wp" {
+  worker_pool "wp" {
```

Comment:
Won't this cause conflict because there will be multiple CI piplelines creating same worker pool ?

Reply:
No, because the worker pool name gets prefixed with the cluster name.